.github/workflows/check-config.yml

```diff
@@ -0,0 +1,18 @@
+on: [ pull_request ]
+
+jobs:
+  check-config:
+    runs-on: ubuntu-latest
+    outputs:
+      targets: ${{ steps.set-matrix.outputs.targets }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: check if config.json is sorted alphabetically
+        run: |
+          jq -r ".[].target" config.json > entries.txt
+          sort entries.txt > entries_sorted.txt
+          if ! output=$(diff -y entries.txt entries_sorted.txt); then
```

Comment:
huh, how does this work? I guess because diff returns an exit code of 1 if there is a diff, and then the parent command sees that exit code too?

Reply:
`diff -y` always returns an output, but we only want to print it when there's actually a diff between the two files.